modules/navier_stokes/src/fvkernels/PINSFVMomentumFriction.C

```diff
@@ -49,9 +58,10 @@ PINSFVMomentumFriction::computeFrictionWCoefficient(const Moose::ElemArg & elem_
 {
   ADReal coefficient = 0.0;
   if (_use_Darcy_friction_model)
-    coefficient += (*_cL)(elem_arg, state)(_index)*_rho(elem_arg, state) / _eps(elem_arg, state);
+    coefficient += (*_cL)(elem_arg, state)(_index)*_rho(elem_arg, state);
```

Comment:
we're still not using the Darcy factor as cL
here 
cL = deltaP / L / rho / U
whereas (from 
[link1](https://holzmann-cfd.com/community/blog-and-tools/darcy-forchheimer)
[link2](https://www.cfd-online.com/Forums/openfoam-community-contributions/222195-darcy-forchheimer-online-calculator.html)
[link3](https://www.simscale.com/knowledge-base/predict-darcy-and-forchheimer-coefficients-for-perforated-plates-using-analytical-approach/) )
fD = deltaP / L / U / mu

our formulation is still correct I think because in the Darcy pre-factor we have:
mu / rho / Dh^2 so 
(our fD) * prefac  =  deltaP/L / rho / U
our fD = (deltaP / L / U / mu) = regular fD (ignoring Dh^2 because I cant figure out where it's supposed to go)

Reply:
Ok so it seems that for the material calculation of the Darcy coefficient material you want me to factor out mu (and into the kernel) and factor in rho (so it will leave the kernel for Darcy).

And then for material calculation of Forchheimer I need to factor in a factor of two (and factor in a factor of 1/2 in the kernel)

This is the last time I'm going to make a change for this. If there are more requests to change what terms are where, I'm just going to move on from this PR

Reply:
man it feels dumb bringing these 1/2 and 2 coefficients in. This is going to cause more bewilderment and confusion

Reply:
> Ok so it seems that for the material calculation of the Darcy coefficient you want me to factor out mu (and into the kernel) and factor in rho (so it will leave the kernel for Darcy).

yes I think that s what people would expect when specifying the Darcy coefficient. 

For Forchheimer I have not redone the math. So I m going to tentatively say yes but also a factor of 2 is not a big deal, we could just tell users to include it in their input.

Reply:
I've reimplemented it to match the Holzmann and SimScale sources